FSI/Force.H

```diff
@@ -56,11 +56,18 @@ public:
         */
     );
 
+    //- Constructor
+    Force
+    (
+        const Foam::fvMesh& mesh,
+        const std::string solverType
+    );
+
     //- Write the displacement values into the buffer
-    void write(double * buffer, bool meshConnectivity, const unsigned int dim);
+    virtual void write(double * buffer, bool meshConnectivity, const unsigned int dim);
```

Comment:
I am not sure if we actually need virtual here. A subclass can just redefine a method, even when it is not virtual. We should still explicitly use a `Stress` object to get the stresses, not a `Force` one.

Reply:
Alright, I will remove it.